core/src/server/web_impl/handler/WebRequestHandler.h

```diff
@@ -0,0 +1,165 @@
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <string>
+#include <utility>
+
+#include <opentracing/mocktracer/tracer.h>
+#include <oatpp/web/server/api/ApiController.hpp>
+
+#include <oatpp/core/data/mapping/type/Object.hpp>
+#include <oatpp/core/macro/codegen.hpp>
+
+#include "server/web_impl/Types.h"
+#include "server/web_impl/dto/CmdDto.hpp"
+#include "server/web_impl/dto/ConfigDto.hpp"
+#include "server/web_impl/dto/DevicesDto.hpp"
+#include "server/web_impl/dto/IndexDto.hpp"
+#include "server/web_impl/dto/PartitionDto.hpp"
+#include "server/web_impl/dto/TableDto.hpp"
+#include "server/web_impl/dto/VectorDto.hpp"
+
+#include "server/context/Context.h"
+#include "server/delivery/RequestHandler.h"
+#include "utils/Status.h"
+
+namespace milvus {
+namespace server {
+namespace web {
+
+#define RETURN_STATUS_DTO(STATUS_CODE, MESSAGE)      \
+    do {                                             \
+        auto status_dto = StatusDto::createShared(); \
+        status_dto->code = (STATUS_CODE);            \
+        status_dto->message = (MESSAGE);             \
+        return status_dto;                           \
+    } while (false);
+
+#define ASSIGN_RETURN_STATUS_DTO(STATUS)                    \
+    do {                                                    \
+        int code;                                           \
+        if (0 != (STATUS).code()) {                         \
+            code = WebErrorMap((STATUS).code());            \
+        } else {                                            \
+            code = 0;                                       \
+        }                                                   \
+        RETURN_STATUS_DTO(code, (STATUS).message().c_str()) \
+    } while (false);
+
+StatusCode
+WebErrorMap(ErrorCode code);
+
+class WebRequestHandler {
+ private:
+    std::shared_ptr<Context>
+    GenContextPtr(const std::string& context_str) {
+        auto context_ptr = std::make_shared<Context>("dummy_request_id");
+        opentracing::mocktracer::MockTracerOptions tracer_options;
+        auto mock_tracer =
+            std::shared_ptr<opentracing::Tracer>{new opentracing::mocktracer::MockTracer{std::move(tracer_options)}};
+        auto mock_span = mock_tracer->StartSpan("mock_span");
+        auto trace_context = std::make_shared<milvus::tracing::TraceContext>(mock_span);
+        context_ptr->SetTraceContext(trace_context);
+
+        return context_ptr;
+    }
+
+ public:
+    WebRequestHandler() {
+        context_ptr_ = GenContextPtr("Web Handler");
+    }
+
+    Status
+    getTaleInfo(const std::shared_ptr<Context>& context, const std::string& table_name,
```

Comment:
getTableInfo?

Reply:
Any doubt ? May the function name is ill-formed